src/components/DateInput/dateInput.stories.mdx

```diff
@@ -42,14 +42,16 @@ prop.
 ## Default selected date
 
 The input element in `DateInput` will be empty by default.
-Pass an instance of `Date` or a parsable date string (`MM/DD/YYYY`) to the `defaultDate` prop to set a default value.
+Pass an instance of `Date` or a parsable date string to the `defaultDate` prop to set a default value.
+Due to inconsistency across browsers with how native `Date` parsing works, we recommend
+passing a string to `defaultDate`.
```

Comment:
Is it worth modifying this so that the description only talks about string dates, and then add a deprecation notice about date objects (that is, if you actually want to deprecate it)? I'm just think that strting the sentence with "Pass an instance of `Date`..." some might read no further.

Reply:
Removing the deprecation down the line sounds like FDS `v10` energy to me 🙃